src/lib/Libifl/int_status.c

```diff
@@ -46,7 +46,132 @@
 
 #include <string.h>
 #include <stdio.h>
+#include <ctype.h>
 #include "libpbs.h"
+#include "pbs_ecl.h"
+#include "libutil.h"
+#include "attribute.h"
+#include "cmds.h"
+#include "pbs_internal.h"
+
+
+extern char * PBS_get_server(char *, char *, uint *);
+
+
+/**
+ * @brief
+ * 	get_available_conn:
+ *	get one of the available connection from multisvr sd
```

Comment:
It looks like this function returns a socket descriptor given a list of server connections. The input is a server instances list not a sd. Please update the description accordingly.